beacon-chain/sync/subscriber.go

```diff
@@ -87,18 +107,23 @@ func (s *Service) registerSubscribers(epoch primitives.Epoch, digest [4]byte) {
 	)
 	if flags.Get().SubscribeToAllSubnets {
 		s.subscribeStaticWithSubnets(
+			"Attestation",
```

Comment:
Can we provide the actual topic or object type here ? Its better than using hardcoded strings here

Reply:
This `description` string is only used in:
```go
log.WithField("digest", digest).Infof("%s subnets with are no longer valid, unsubscribing from all of them.", description)
```

Before, we had the log hardcoded version:

```go
log.Warnf("Attestation subnets with digest %#x are no longer valid, unsubscribing from all of them.", digest)
```

So, it was already hard-coded before, but not at the same location.
I can for sure use the actual topic instead, but the log will be less human readable.

Reply:
Could you possible use a helper function with a switch of all topics to derive this (ex: If the topic is the attestation subnet, then the description will always be `Attestation` . And so on. This way we don't need to add in an additional argument